### ISSUE TITLE
add support for inches

### ISSUE THREAD

**Issue by mglotzbach** [sir-buckyball/chrome-gcode-sender#46]
Would be good if support for inches could be added.  I am thinking that the same controls can be used and simple code update can be made.  stepSize >= 1 can use same logic as mm giving user ability to step in 1 in, 10in increments.  For stepSize < 0, the getStepSize method can be Math.pow(2, $scope.stepSize).  This would give very convenient increments in nominal woodworking units 1/4, 1/8, 1/16, 1/32, etc...

I have branched master to develop this.  More than happy to finish and submit if you are willing to accept.

**Comment by sir-buckyball** [comment]
Yes I'd be willing to review pull requests (may take a few days due to limited free time).

I think having an option to choose inches/mm in the settings panel would make sense. As for the units; I'd prefer using logical/reasonable step sizes instead of trying to stick to powers (like the current code does).
